app/routers/event.py

```diff
@@ -21,15 +33,64 @@ async def eventedit(request: Request):
                                       {"request": request})
 
 
+@router.post("/edit")
+async def create_new_event(request: Request, session=Depends(get_db)):
```

Comment:
This function is a bit too long - can you think of a way to split it?

Reply:
Please don't close review comments that you haven't addressed yet.
If you don't agree with the comment - please explain why, we can talk about it :)

Marking the comment as resolved makes me go through the whole code again.